addons/mass_mailing/controllers/main.py

```diff
@@ -162,3 +162,51 @@ def send_feedback(self, mailing_id, res_id, email, feedback, token):
                 record.sudo().message_post(body=_("Feedback from %s: %s" % (email, feedback)))
             return bool(records)
         return 'error'
+
+    @http.route(['/mail/mailing/<int:mailing_id>/browser_view'], type='http', website=True, auth='public')
```

Comment:
We will gradually update convention for route naming. Here we could simply define
```
/mailing/<int:mailing_id>/view
```
We gain the /mail prefix (unnecessary) and view is sufficient. Reason is that we want to have a single prefix based on model (mailing), then id, then action.